packages/gasket-core/lib/index.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-console, no-process-env */
+// @ts-check
+/// <reference types="./index" />
+
+import GasketEngine from '@gasket/engine';
+import { applyConfigOverrides } from '@gasket/utils';
+
+/**
+ * Get the environment to use for the gasket instance.
+ * Defaults to `development`.
+ * @returns {string} env
+ */
+function getEnvironment(
+  // flags, commandId, warn
```

Comment:
Commented code not sure if it's intentional

Reply:
I left these commented out as there might still be a scenario where commands and flags could come into play. I'll add a todo so we remember to clean them up and implement - I don't want to loose track of them for now